aicsimageio/readers/ome_zarr_reader.py

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from typing import Any, Dict, List, Optional, Tuple, Union
+
+import xarray as xr
+from fsspec.implementations.local import LocalFileSystem
+from fsspec.spec import AbstractFileSystem
+
+from .. import exceptions, types
+from ..dimensions import DimensionNames
+from ..metadata import utils as metadata_utils
+from ..utils import io_utils
+from .reader import Reader
+
+try:
+    from ome_zarr.io import parse_url
+    from ome_zarr.reader import Reader as ZarrReader
+
+except ImportError:
+    raise ImportError(
+        "ome_zarr is required for this reader. " "Install with `pip install 'ome_zarr'`"
+    )
+
+###############################################################################
+
+
+class OmeZarrReader(Reader):
+    """
+    Wraps the ome-zarr-py API to provide the same aicsimageio Reader API but for
+    OmeZarr images.
+
+    Parameters
+    ----------
+    image: types.PathLike
+        Path to image file to construct Reader for.
+    fs_kwargs: Dict[str, Any]
```

Comment:
You have an optional parameter `channel_names` that would be good to include here too

Reply:
parameter removed with [9c2ccf6](https://github.com/AllenCellModeling/aicsimageio/pull/493/commits/9c2ccf6506f33867fc773355c52a38d6c01b6c0e)